src/transformers/pipelines/image_text_to_text.py

```diff
@@ -445,8 +457,13 @@ def postprocess(
             ]
 
         # Decode inputs and outputs the same way to remove input text from generated text if present
-        generated_texts = self.processor.post_process_image_text_to_text(generated_sequence, **postprocess_kwargs)
-        decoded_inputs = self.processor.post_process_image_text_to_text(input_ids, **postprocess_kwargs)
+        skip_special_tokens = skip_special_tokens if skip_special_tokens is not None else True
+        generated_texts = self.processor.post_process_image_text_to_text(
```

Comment:
`skip_special_tokens` was already an arg to this function :)

Reply:
yep along with tokenizer kwargs!